src/jockey/stage.rs

```diff
@@ -173,6 +176,25 @@ impl Stage {
                     }
                 };
 
+                let gltf = match object.get("gltf") {
+                    Some(s) => match s.as_str() {
+                        Some(value) => {
+                            match geometry_from_gltf(value.to_string()) {
+                                Ok(s) => Some(s),
+                                Err(e) => {
+                                    log::warn!("{}", e);
```

Comment:
This should probably be a hard error rather than a warning, consider using `return Err(format!("{}", e));` instead

Reply:
ahhh Stage's from_yaml is designed to return errors! didn't noticed to that